distdir_deps.bzl

```diff
@@ -11,544 +11,65 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-"""List the distribution dependencies we need to build Bazel.
 
-Note for Bazel users: This is not the file that you are looking for.
-This is internal source and is not intended to tell you what version
-you should use for each dependency.
-"""
-
-load("//src/tools/bzlmod:utils.bzl", "get_canonical_repo_name")
-
-##################################################################################
-#
-# The list of repositories required while bootstrapping Bazel offline
-#
-##################################################################################
-DIST_ARCHIVE_REPOS = [get_canonical_repo_name(repo) for repo in [
-    "abseil-cpp",
-    "apple_support",
-    "bazel_skylib",
-    "blake3",
-    "c-ares",
-    "com_github_grpc_grpc",
-    "com_google_protobuf",
-    "io_bazel_skydoc",
-    "platforms",
-    "rules_cc",
-    "rules_go",
-    "rules_java",
-    "rules_jvm_external",
-    "rules_license",
-    "rules_pkg",
-    "rules_proto",
-    "rules_python",
-    "upb",
-    "zlib",
-    "zstd-jni",
-]] + [(get_canonical_repo_name("com_github_grpc_grpc") + suffix) for suffix in [
-    # Extra grpc dependencies introduced via its module extension
-    "~grpc_repo_deps_ext~bazel_gazelle",  # TODO: Should be a bazel_dep
-    "~grpc_repo_deps_ext~bazel_skylib",  # TODO: Should be removed
-    "~grpc_repo_deps_ext~com_envoyproxy_protoc_gen_validate",
-    "~grpc_repo_deps_ext~com_github_cncf_udpa",
-    "~grpc_repo_deps_ext~com_google_googleapis",
-    "~grpc_repo_deps_ext~envoy_api",
-    "~grpc_repo_deps_ext~rules_cc",  # TODO: Should be removed
-]]
+"""WORKSPACE default repository definitions."""
```

Comment:
IIUC there's no reason this file should be called 'distdir_deps' anymore, right? It only contains information about WORKSPACE suffix repos, which are (for example) not used by Bzlmod builds at all.

Reply:
Make sense, I renamed it to `workspace_deps.bzl`